test/WebJobs.Script.Tests.Integration/Host/StandbyModeTests.cs

```diff
@@ -116,6 +116,7 @@ public void EnsureInitialized_PlaceholderMode()
                 _webHostResolver.EnsureInitialized(settings);
 
                 _settingsManager.SetSetting(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "0");
+                _settingsManager.SetSetting(EnvironmentSettingNames.AzureWebsiteContainerReady, "1");
```

Comment:
Existing tests are all verifying the requirement for this new value to be present now to trigger specialization